### DISCUSSION TITLE
Table Schema doesn't update using custom query and transformer

### DISCUSSION THREAD

**Discussion by wipplingermax** [Budibase/budibase#11129] [Help] [answered]
Hey folks,

I'm struggeling really long with the following behaviour:

I have a custom query with a transformer that transforms data in the following way: 
```
[
  {
    "name": "entry1",
    "a":1,
    "b":2,
  }, 
  {
    "name": "entry2",
    "a":3,
    "b":4,
  } 
]
```

to 

```
[
  {
    "property": "a",
    "entry1": "1"
    "entry2": "3",
  },
  {
    "property": "b",
    "entry1": "2"
    "entry2": "4",
  }
]
```

The problem is that the number of properties varies and the table can have different numbers of columns.

In the custom query the result is updated, but not on the screen/table.

Does anyone have an idea?

**Comment by R2bEEaton** [comment]
Did you test and save your custom query? Also, make sure you publish your changes.

**Comment by melohagan** [comment, marked as answer]
Hey @wipplingermax 

>  the table can have different numbers of columns

We currently don't support dynamic schemas. The schema is determined from the response of your query, and set when `Save` is clicked.

There is an open issue for handling dynamic form inputs: https://github.com/Budibase/budibase/issues/5658
